### ISSUE TITLE
[Question] How can I use the substring function in the "Use label function" field

### ISSUE THREAD

**Issue by OzyOzk** [thingsboard/thingsboard#7318]
**Component**

 * UI

**Description**
I want to return a substring of the entityName in my label function. How can I JavaScript's substring in the html field? I've tried adding the script in a <script> tag but this causes the label to not render. I'm currently using the following function;

`return '<body> <span style="color:blue;", id="myLabel">${entityName}</span></body>';`

**Environment**

 * OS:  Windows 10
 * ThingsBoard: cloud
 * Browser: FireFox

**Comment by devaskim** [comment]
```
let newName = entityName.substring(0, Math.floor(entityName.length / 2));
return '<body> <span style="color:blue;", id="myLabel">${newName }</span></body>';
```

**Comment by OzyOzk** [comment]
> 

This doesn't work unfortunately.

**Comment by OzyOzk** [comment]
> Note: use ` instead of '
> 
> ```
> let newName = entityName.substring(0, Math.floor(entityName.length / 2));
> return `<body> <span style="color:blue;", id="myLabel">${newName}</span></body>`;
> ```

This doesn't work unfortunately. The label does not render in the image map widget when I try to use a local variable inside the html template literal.

**Comment by devaskim** [comment]
Did you try to do same as I suggested but without local variable? And are you sure the function is being called, i.e. is label blue-colored?
```
return `<body> <span style="color:blue;", id="myLabel">${entityName.substring(0, Math.floor(entityName.length / 2))}</span></body>`;
```

**Comment by OzyOzk** [comment]
> `` return `<body> <span style="color:blue;", id="myLabel">${entityName.substring(0, Math.floor(entityName.length / 2))}</span></body>`; ``

Yes, the function does get called. It does come back blue but when I add the substring code it stops working, even without a var.
